package.json

```diff
@@ -19,15 +19,25 @@
     "David Giles <david_giles@ultimatesoftware.com>"
   ],
   "license": "MIT",
+  "standard": {
+    "env": [
+      "jasmine"
+    ]
+  },
   "dependencies": {
     "@google/maps": "^0.4.5",
+    "amqp": "^0.2.7",
+    "amqplib": "^0.5.2",
     "eureka-js-client": "^4.4.1",
+    "fix": "0.0.6",
     "ip": "^1.1.5",
-    "npm": "^5.7.1",
+    "jasmine": "^3.1.0",
+    "npm": "^6.2.0",
```

Comment:
I don't think you need to install npm per package, if you need to specify a version of npm it should use the engines property: 
https://docs.npmjs.com/files/package.json#engines